### PR TITLE
Cleaned in module code

### PR DESCRIPTION
**Pull request by moncef-essid**
copy/paste error

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | A little cleaning in module code, no need to use `country->id`
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Make sure that you can manage two modified settings in module

### CODE REVIEW

**Comment by Arman-Hosseini** [comment]
Any reason?

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Please correctly fill the pull request template

**Comment by kpodemski** [comment]
This is of course is a good change, but please follow instructions from @PierreRambaud and provide us with content of PR template. Could you do that?

File: ps_emailsubscription.php [hunk 1]
```diff
@@ -1119,7 +1119,7 @@ public function renderExportForm()
                         'hint' => $this->trans('Customers can subscribe to your newsletter when registering, or by entering their email in the newsletter form.', array(), 'Modules.Emailsubscription.Admin'),
                         'name' => 'SUSCRIBERS',
                         'required' => false,
-                        'default_value' => (int) $this->context->country->id,
+                        'default_value' => 0,
```
Comment (kpodemski) [inline ps_emailsubscription.php:1122]:
```suggestion
                        'default_value' => 1,
```

File: ps_emailsubscription.php [hunk 2]
```diff
@@ -1138,7 +1138,7 @@ public function renderExportForm()
                         'hint' => $this->trans('Partner offers subscribers have agreed to receive your partners\' offers.', array(), 'Modules.Emailsubscription.Admin'),
                         'name' => 'OPTIN',
                         'required' => false,
-                        'default_value' => (int) $this->context->country->id,
+                        'default_value' => 0,
```
Comment (kpodemski) [inline ps_emailsubscription.php:1141]:
```suggestion
                        'default_value' => 1,
```

**Review by kpodemski — CHANGES_REQUESTED** [review]

**Comment by kpodemski** [comment]
@PierreRambaud 

you can check this PR again, I help @moncef-essid as we don't have any reply from him

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @moncef-essid,

It is ok :heavy_check_mark: 
I checked with develop branch & 177x branch.
https://drive.google.com/file/d/1LtJw5krKkWmZgBraLbjVR7bZgwIArI13/view?usp=sharing

Thank you!

**Comment by Progi1984** [comment]
Thanks @moncef-essid & @khouloudbelguith

**Pull request merged**